admin/management/views.py

```diff
@@ -120,7 +120,14 @@ def post(self, request, *args, **kwargs):
         if monthly_report_date:
             report_date = isoparse(monthly_report_date).date()
         else:
-            report_date = datetime.datetime.now().date()
+            messages.error(request, 'Monthly Reporter can\'t be run in future or current month')
+            return redirect(reverse('management:commands'))
```

Comment:
I would recommend just using `None` in the `else` case e.g.
```
if monthly_report_date:
    report_date = isoparse(monthly_report_date).date()
else:
    report_date = None
```
This is because the [default behavior](https://github.com/CenterForOpenScience/osf.io/blob/decc312a2be6111f1faccfd708d137a5d69343a7/osf/management/commands/monthly_reporters_go.py#L25-L30) of the `monthly_reporters_go` scripts, when year/month are `None`, is to use the prior month (i.e. if triggering it on October 11th without specifying month, it would run the job for the month of September).